localstack/utils/container_utils/container_client.py

```diff
@@ -103,6 +105,24 @@ class DockerPlatform(str):
     linux_arm64 = "linux/arm64"
 
 
+@dataclasses.dataclass
+class Ulimit:
```

Comment:
I guess we aim to keep the `container_client` generic and therefore added a Ulimit data class instead of importing `docker.types.Ulimit`